### PR TITLE
objectives for morph and nightmare

### PR DESCRIPTION
**Pull request by ynot01**
### Intent of your Pull Request

gives generic objectives + stay alive to morph and nightmare
also tells all ghosts morph is antag
some cool greet sounds too

#### Changelog

:cl:  
rscadd: Added objectives for morph
soundadd: Added greeting sound for morph
rscadd: Added objectives for nightmare
soundadd: Added greeting sound for nightmare
tweak: Morphs now show_to_ghosts
/:cl:

### CODE REVIEW

**Comment by Mqiib** [comment]
Yes let's give the midround antag that already murderbones but could be way cooler an official pass to just murderbone and do nothing else. Good idea, very cool.

**Comment by ynot01** [comment]
> Yes let's give the midround antag that already murderbones but could be way cooler an official pass to just murderbone and do nothing else. Good idea, very cool.

that's the plan

File: code/modules/antagonists/morph/morph_antag.dm [hunk 1]
```diff
@@ -2,5 +2,21 @@
...
 	show_in_antagpanel = FALSE
+	show_to_ghosts = TRUE
 
-//It does nothing! (Besides tracking)
\ No newline at end of file
+/datum/antagonist/morph/proc/forge_objectives()
+	var/datum/objective/new_objective = new
+	new_objective.owner = owner
+	objectives += new_objective
+	new_objective.explanation_text = "Consume everything." // you can do whatever
```
Comment (Amelia0010) [inline code/modules/antagonists/morph/morph_antag.dm:12]: should probably make the objective auto-complete if there's no check for completing it

File: code/modules/antagonists/nightmare/nightmare.dm [hunk 2]
```diff
@@ -3,3 +3,20 @@
...
 	show_name_in_check_antagonists = TRUE
 	show_to_ghosts = TRUE
+
+/datum/antagonist/nightmare/proc/forge_objectives()
+	var/datum/objective/new_objective = new
+	new_objective.owner = owner
+	objectives += new_objective
+	new_objective.explanation_text = "<span class='userdanger'>Kill.</span>" // you can do whatever
+	var/datum/objective/survive/survival = new
+	survival.owner = owner
```
Comment (Amelia0010) [inline code/modules/antagonists/nightmare/nightmare.dm:14]: see above

**Review by Amelia0010 — CHANGES_REQUESTED** [review]

**Comment by Mqiib** [comment]
I'd rather nightmares have no objectives than haha funny kill everyone but bold.

**Review by Amelia0010 — APPROVED** [review]

**Pull request merged**